src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_quantization.cpp

```diff
@@ -588,6 +588,10 @@ void prepare_quantization::prepare_asymmetric_quantization(program &p, convoluti
         in1.get_output_layout().data_type != data_types::i8)
         return;
 
+    const size_t feature_idx = 1;
+    if (asymmetric_data && in0.get_output_layout().get_partial_shape()[feature_idx].is_dynamic())
+        OPENVINO_ASSERT(false, "[GPU] Can't support prepare_quantization when feature is dynamic.");
```

Comment:
Not sure if there are cases where input feature dimension of convolution node is dynamic, but can't we proceed without this assertation and optimization? I don't think this optimization is strictly required 

Could you please add related tests?

Reply:
Yes, I agree, no need the assertion in here and Mingyu also agreed.  I'll restore 'return'.